modules/ingest-geoip/src/yamlRestTest/resources/rest-api-spec/test/ingest_geoip/30_geoip_stats.yml

```diff
@@ -0,0 +1,10 @@
+---
+"Test geoip stats":
+  - do:
+      ingest.geo_ip_stats: {}
+  - match: { stats.successful_downloads: 0 }
```

Comment:
I think this may fail if during the yaml tests execution the geoip downloader actually downloads databases?
Perhaps we should just validate the existence of specific fields (using `is_true`)?

Reply:
it shouldn't downloader is disabled for these tests but it's perhaps safer to use `is_true` as you suggested, I'll update it

Reply:
I used `gte` - `is_true` doesn't work with 0 ( it treats it as `false`)

Reply:
:( too bad that there isn't another way to check the existence of a field.